src/components/AccountSettings.tsx

```diff
@@ -1,330 +1,277 @@
     }
+  } catch (err: any) {
+    displayToast(err.reason as string, 'danger')
+  }
+}
 
-    const ProfileValidationState = (verificationStateEnum: VerificationStateEnum) => {
-        if(verificationStateEnum === 'SUBMITTED'){
-            return <div className='verified-img-container'><img className="BadgeIcon" src={CheckmarkBadgeDark} alt='CheckmarkBadgeDark'/>Pending Verification</div>
-        } else if(verificationStateEnum === 'REJECTED' || verificationStateEnum ==='SUSPENDED'){
-            return (
-                <div className='ValidationStateContainer'>
-                    <Typography variant='headline3'>
-                        {verificationStateEnum === 'REJECTED' ? REJECTED_TEXT : SUSPENDED_TEXT}
-                    </Typography>
-                    <Typography variant='body1' >
-                        {verificationState?.reason}
-                    </Typography>
-                    <VerifyButton/>
-                </div>
-            )
-        } else {
-            return <VerifyButton/>
-        }
-    }
+  useEffect(() => {
+    getUserData()
+  }, []
+  )
 
-    // Closes any forms and resets the fields.
-    const cancelEdit = () => {
-        setEditUsername(false)
-        setUpdatedUsername('')
-        setChangePW(false)
-    }
+  const handleScroll = (ref:React.RefObject<HTMLDivElement>) => {
+    ref.current?.scrollIntoView({behavior:'smooth'})
+  }
 
-    interface EditFieldProps {
-        label: string, 
-        updatedValue: string, 
-        updateFn: Function,
-    }
-    
-    // Component to edit fields (username / email)
-    const EditField: React.FC<EditFieldProps> = ({label, updatedValue, updateFn}) => {
-        return(
-            <div>
-                <FormGroup className='required'>
-                    <FormLabel>{label}</FormLabel>
-                    <FormControl
-                        onChange={e => updateFn(e.target.value)}
-                        style={{maxWidth:'320px'}}
-                        value = {updatedValue}/>
-                    <Button className='btn-container emptyButton' onClick={cancelEdit}>
-                        Cancel
-                    </Button>
-                    <Button className='btn-container'  variant='secondary' onClick={onUpdateUserProfile}>
-                        Save Changes
-                    </Button>
-                </FormGroup>
+  return (
+    <div className='panel-wrapper-bg with-account-setting'>
+      <nav className='account-setting-panel nav-panel'>
+        <Button onClick={() => handleScroll(profileInformationRef)} variant='outlined'>Profile Information</Button> 
+        <Button onClick={() => handleScroll(changePasswordRef)} variant='outlined'>Change Password</Button>
+        <Button onClick={() => handleScroll(trustCredentialRef)} variant='outlined'>Trust & Credentials</Button>
+        <Button onClick={() => handleScroll(personalAccessTokenRef)} variant='outlined'>Personal Access Tokens</Button>
+      </nav>
```

Comment:
Functionality works for scrolling to specific parts of the page. However, it would be ideal if the nav bar would stick to its position.